app/views/controllers/projects/violations/edit.html.erb

```diff
@@ -0,0 +1,31 @@
+<%
+  add_page_title(:violations_edit_title.t)
+  @container = :wide
+%>
+
+<h4 class="pb-3">
+  <%= :PROJECT.l + ": " + link_to_object(@project).t %>
+<h4>
+
+<div class="smaller">
+
+<%= help_block(:div, violations_help_text(@project)) %>
+
+<%=
+  form_with(
+    model: @project, url: project_violations_path, method: :post
+  ) do |form|
+%>
```

Comment:
@nimmolo: Should stuff above be included in the form?

Reply:
I kind of like the help block being outside the form element. 

I'd give the form an HTML ID though, for testing, maybe "project_violations_form", and that should allow you to skip the inner div#violations (not sure)

Reply:
Thanks for suggesting that I `id` the form rather than the div. It's done and I will resolve the conversation.